app/client/components/organizationWrapper.jsx

```diff
@@ -12,9 +12,9 @@ const OrganizationWrapper = Component =>
       let query_params = parseQuery(location.search);
       if (
         query_params["org"] &&
-        this.props.login.organisationId !== query_params["org"]
+        this.props.login.organisationId !== parseInt(query_params["org"])
       ) {
-        let org_id = query_params["org"];
+        let org_id = parseInt(query_params["org"]);
```

Comment:
`organisationId` is defined as a `number` in `auth/types`

https://github.com/teamsempo/SempoBlockchain/blob/efaf2c66df4f0333c2015bbcfd31c320b261d92f/app/client/reducers/auth/types.ts#L11-L13

https://github.com/teamsempo/SempoBlockchain/blob/efaf2c66df4f0333c2015bbcfd31c320b261d92f/app/client/reducers/auth/types.ts#L35